CHANGELOG.rst

```diff
@@ -45,6 +45,10 @@ Changed
 - Updated Mattermost connector documentation to be more clear.
 - Updated format strings to f-strings where appropriate.
 - Updated tensorflow requirement to ``1.15.0``
+- Updated ``/status`` endpoint to show correct path to model archive
```

Comment:
please move this to the correct position

Reply:
sorry where should this go again?  I had this originally there from when we pushed the first changes so should I just move this up or say added tests in under the most recent changelog section?